docs/src/development/local/_index.md

```diff
@@ -1,151 +1,94 @@
-|   march-tests      | march tests      | Active | development |
-|   april-release    | april release    | Active | development |
-|   april-tests      | april tests      | Active | development |
-+--------------------+------------------+--------+-------------+
-```
-
-### Launch the download
-
-If your app's code isn't already present locally on your computer,
-you can download a copy of it.
-To do so, move to the directory where you want to save a copy of your project 
-and run this command:
-
-```bash
-platform get {{< variable "PROJECT_ID" >}}
-```
-
-When prompted, choose a name for the target project directory and press **Enter**.
-You get output similar to the following:
-
-```
-Downloading [project-name](abcdefgh1234567)
-  Cloning into '/Users/username/project-directory'...
-  remote: counting objects: 3318, done.        
-  Receiving objects: 100% (3318/3318), 20.23 MiB | 3.71 MiB/s, done.
-  Resolving deltas: 100% (568/568), done.
-
-The project [project-id](abcdefgh1234567) was successfully downloaded to: 'project-directory'
-```
-
-You can now access your app's code from the project directory created on your computer.
-Note that inside the project directory, the `.platform/local` subdirectory is excluded from Git. 
-It contains builds and any local metadata about your project the [Platform.sh CLI](../../administration/cli/_index.md) needs.
+You need to have:
+
+- A Platform.sh account:
+  new users can sign up for a [free trial account](https://auth.api.platform.sh/register)
+- A working project,
+  either started from a [template](../../development/templates.md) 
+  or with your own code pushed to Platform.sh
+- [Git](https://git-scm.com/downloads)
+- The [Platform.sh CLI](../../administration/cli/_index.md)
+
+## 1. Get your code
+
+If you don't have your app code on your computer, download a copy.
+
+1.  Get your project ID by running `platform projects`.
+
+2.  Get the code by running the following command:
+
+    ```bash
+    platform get {{< variable "PROJECT_ID" >}} {{< variable "TARGET_DIRECTORY_NAME" >}}
+    ```
+
+    Or pull from your [integrated Git repository](../../integrations/source/_index.md).
+
+You can now access your code from the project directory on your computer.
+The CLI created a `.platform/local` directory that's excluded from Git. 
+It contains builds and local metadata about your project.
 
 You can now make changes to your project without pushing to Platform.sh each time to test them. 
 Instead, you can locally build your application using the Platform.sh CLI.
 
 Note that if your app contains services, you need to open an SSH tunnel to connect to them.
 For more information, see how to [connect services](../../add-services#2-connect-the-service).
 
-## 2. Build your site locally
+## 2. Connect to services
+
+If your app requires services to run, you have two options for developing locally:
+
+- [Tethered local development](./tethered.md) involves running your app on a local web server
```

Comment:
Wow, I had no idea. I certainly wouldn't recommend this to people, would suggest that this be offered as a second, distant alternative, rather than offering it first.